css/selectors/any-link-dynamic-001.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS test: Handling of dynamic changes to :any-link selectors</title>
+<link rel="author" title="Boris Zbarsky" href="mailto:bzbarsky@mit.edu">
+<link rel="match" href="any-link-dynamic-001-ref.html">
+<link rel="help" href="https://drafts.csswg.org/selectors-4/#the-any-link-pseudo">
+<style>
+  span { color: green; }
+  :any-link + span { color: red; }
+</style>
+<body onload="window.oldColor = getComputedStyle(document.querySelector('span')).color;
```

Comment:
@emilio this seems unnecessary

Reply:
This was just moving the test around, but I agree. It doesn't hurt, and it just wants to make sure that the style of the span has been computed before, but feel free to remove that line if you want, or I can do that too.